### PR TITLE
feat: [NMP-26] Feature** Created basic labeller action

### PR DESCRIPTION
**Pull request by kcaparas1630**
# Description

Created a basic labeller action based on the directories in our current project.

## Type of changes

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## Checklist

- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have already been accepted and merged

## Further comments

---

Thanks for the PR!

Deployments, as required, will be available below:
- [Frontend](https://nr-sustainment-capstone-2024-59-frontend.apps.silver.devops.gov.bc.ca)
- [Backend](https://nr-sustainment-capstone-2024-59-frontend.apps.silver.devops.gov.bc.ca/api)

Please create PRs in draft mode.  Mark as ready to enable:
- [Analysis Workflow](https://github.com/bcgov/nr-sustainment-capstone-2024/actions/workflows/analysis.yml)

After merge, new images are deployed in:
- [Merge Workflow](https://github.com/bcgov/nr-sustainment-capstone-2024/actions/workflows/merge.yml)

### CODE REVIEW

**Review by GDamaso — COMMENTED** [review]
Paths are correct

File: .github/labeller.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+'Environment Variables':
+  - '/**/*.env*/'
+
+# Any changes to any swagger documents
+'Swagger':
+  - /backend/src/docs/*.yml
+
+# Any changes to any swagger configurations
+'Swagger Configurations':
+  -/backend/src/config/*.ts
```
Comment (LocalNewsTV) [inline .github/labeller.yml:32]:
```suggestion
  - /backend/src/config/*.ts
```

Comment (GDamaso) [reply .github/labeller.yml:32]: Good eyes!
Shouldn't we expand the linter and formatter to be running on the root dir files too?

Comment (LocalNewsTV) [reply .github/labeller.yml:32]: More trouble than its worth
Your root files will seldomly change after setup (besides the README), and your linter wouldn't catch this example anyways.

Comment (kcaparas1630) [reply .github/labeller.yml]: All done!

**Review by LocalNewsTV — APPROVED** [review]

**Pull request merged**